cmd/mt-index-cat/main.go

```diff
@@ -159,11 +161,17 @@ func main() {
 	}
 
 	var cutoff uint32
+	var cutoffMin int64
 	if maxAge != "0" {
 		maxAgeInt, err := dur.ParseNDuration(maxAge)
 		perror(err)
 		cutoff = uint32(time.Now().Unix() - int64(maxAgeInt))
 	}
+	if minAge != "0" {
+		minAgeInt, err := dur.ParseNDuration(minAge)
+		perror(err)
+		cutoffMin = int64(time.Now().Unix() - int64(minAgeInt))
```

Comment:
would it make sense to only call `time.Now().Unix()` once and then reuse the returned value for the calculation of `cutoff` and `cutoffMin`. I know that in reality they're probably only going to be microseconds apart, but I think it's kind of weird that theoretically it would be possible that the time-range which is returned by this utility is not exactly `cutoff - cutoffMin`.

Reply:
yep